batch/test/test_batch.py

```diff
@@ -1464,16 +1464,19 @@ def test_job_private_instance_cancel(client: BatchClient):
     j = bb.create_job(DOCKER_ROOT_IMAGE, ['true'], resources=resources)
     b = bb.submit()
 
-    delay = 0.1
+    tries = 0
     start = time.time()
     while True:
         status = j.status()
         if status['state'] == 'Creating':
             break
         now = time.time()
+
+        tries += 1
+        delay = delay_ms_for_try(tries)
         if now + delay - start > 60:
             assert False, str((status, b.debug_info()))
-        delay = sync_sleep_and_backoff(delay)
+        time.sleep(delay)
```

Comment:
can you use `sync_sleep_before_try here` here?

Reply:
I can't, I need to know the delay so I can ensure we do no more than 60s of delay. This test is different from everywhere else that we use this stuff. It wants to ensure we wait no more than 60s for cancellation.

Reply:
I actually had to add an extra change to this test to ensure we at least wait 60s. This new code seems to back off so quickly that there's a decent chance that our cumulative delay is like ~30s and our next delay is ~30s so the `assert False` is trigged.

Reply:
Ah, missed that. Alright